<ui version="4.0">
 <class>SearchParamsDialog</class>
 <widget class="QDialog" name="SearchParamsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры района поиска</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_main">
      <attribute name="title">
       <string>Основные</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_main">
       <item>
        <widget class="QGroupBox" name="groupBox_id">
         <property name="title">
          <string>Идентификация района</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_id">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_district">
            <item>
             <widget class="QLabel" name="label_district">
              <property name="text">
               <string>Номер района поиска:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_district">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_prefix">
            <item>
             <widget class="QLabel" name="label_prefix">
              <property name="text">
               <string>Префикс подрайонов:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_prefix">
              <property name="text">
               <string>A</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_tech">
         <property name="title">
          <string>Технические параметры</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_tech">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_sru">
            <item>
             <widget class="QLabel" name="label_sru">
              <property name="text">
               <string>Средство определения местоположения SRU:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmb_sru">
              <item>
               <property name="text">
                <string>GNSS</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Радар</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Визуально</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ADF</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Другое</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_accuracy">
            <item>
             <widget class="QLabel" name="label_accuracy">
              <property name="text">
               <string>Точность определения (морские мили):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spin_accuracy">
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="maximum">
               <double>50.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_time">
         <property name="title">
          <string>Временные параметры</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_time">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_start">
            <item>
             <widget class="QLabel" name="label_start_time">
              <property name="text">
               <string>Дата и время начала операции (UTC):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="datetime_start">
              <property name="displayFormat">
               <string>dd.MM.yyyy hh:mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_incident">
            <item>
             <widget class="QLabel" name="label_incident_time">
              <property name="text">
               <string>Время аварии (UTC):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="datetime_incident">
              <property name="displayFormat">
               <string>dd.MM.yyyy hh:mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_duration">
            <item>
             <widget class="QLabel" name="label_duration">
              <property name="text">
               <string>Планируемая продолжительность поиска:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_duration_hours">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>72</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_hours">
              <property name="text">
               <string>час</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_duration_min">
              <property name="maximum">
               <number>59</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_min">
              <property name="text">
               <string>мин</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_drift">
      <attribute name="title">
       <string>Дрейф</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_drift">
       <item>
        <widget class="QGroupBox" name="groupBox_wind">
         <property name="title">
          <string>Ветровые условия</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_wind">
          <item>
           <widget class="QCheckBox" name="chk_use_wind">
            <property name="text">
             <string>Учитывать влияние ветра</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_wind">
            <item>
             <widget class="QLabel" name="label_wind_speed">
              <property name="text">
               <string>Скорость (м/с):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spin_wind_speed">
              <property name="maximum">
               <double>50.000000000000000</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_wind_direction">
              <property name="text">
               <string>Направление (°):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_wind_direction">
              <property name="maximum">
               <number>359</number>
              </property>
              <property name="value">
               <number>270</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_current">
         <property name="title">
          <string>Течения</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_current">
          <item>
           <widget class="QCheckBox" name="chk_use_current">
            <property name="text">
             <string>Учитывать влияние течения</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_current">
            <item>
             <widget class="QLabel" name="label_current_speed">
              <property name="text">
               <string>Скорость (узлы):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spin_current_speed">
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_current_direction">
              <property name="text">
               <string>Направление (°):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_current_direction">
              <property name="maximum">
               <number>359</number>
              </property>
              <property name="value">
               <number>90</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_object">
         <property name="title">
          <string>Характеристики объекта поиска</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_object">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_object_type">
            <item>
             <widget class="QLabel" name="label_object_type">
              <property name="text">
               <string>Тип объекта:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmb_object_type">
              <item>
               <property name="text">
                <string>Человек в воде</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Спасательный плот (4 чел.)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Спасательный плот (6 чел.)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Спасательный плот (10 чел.)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Спасательная шлюпка</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Малое судно</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Обломки (легкие)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Обломки (тяжелые)</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_calculations">
      <attribute name="title">
       <string>Расчеты</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_calc">
       <item>
        <widget class="QLabel" name="label_error_calc">
         <property name="text">
          <string>Расчет суммарной вероятной погрешности:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="txt_error_calc">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_calc_buttons">
         <item>
          <widget class="QPushButton" name="btn_calc_drift">
           <property name="text">
            <string>Рассчитать дрейф</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_calc_probability">
           <property name="text">
            <string>Рассчитать вероятность</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_calc_area">
           <property name="text">
            <string>Рассчитать площадь</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_calc_results">
         <property name="text">
          <string>Результаты расчетов:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="txt_calc_results">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_main_buttons">
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_preview">
       <property name="text">
        <string>Предварительный просмотр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_build">
       <property name="text">
        <string>Построить район</</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>